apps/designer/app/designer/features/sidebar-left/panels/assets/assets.tsx

```diff
@@ -2,20 +2,21 @@ import { ImageIcon } from "@webstudio-is/icons";
 import { ImageManager } from "~/designer/shared/image-manager";
 import { TabName } from "../../types";
 import { Header, CloseButton } from "../../lib/header";
+import { Flex } from "@webstudio-is/design-system";
 
 export const TabContent = ({
   onSetActiveTab,
 }: {
   onSetActiveTab: (tabName: TabName) => void;
 }) => {
   return (
-    <>
+    <Flex direction="column" css={{ height: "100%" }}>
```

Comment:
`flex: 1` probably having its flex children?

Reply:
you mean to try flex: 1 instead of height?

Reply:
yes

Reply:
tried, didn't work